view/theme/frio/templates/profile/schedule.tpl

```diff
@@ -0,0 +1,24 @@
+<div class="generic-page-wrapper">
+	{{* include the title template for the settings title *}}
+	{{include file="section_title.tpl" title=$title}}
+	<form action="profile/{{$nickname}}/schedule" method="post" autocomplete="off">
+		<input type='hidden' name='form_security_token' value='{{$form_security_token}}'>
+		<table id='application-block' class='table table-condensed table-striped'>
+			<thead>
+				<tr>
+					<th>{{$scheduled_at}}</th>
+					<th>{{$content}}</th>
+				</tr>
+			</thead>
+			<tbody>
+				{{foreach $schedule as $row}}
+				<tr>
+					<td>{{$row.scheduled_at}}</td>
+					<td>{{$row.content}}</td>
+					<td><a href="{{$baseurl}}/profile/{{$nickname}}/schedule/delete/{{$row.id}}?t={{$form_security_token}}" class="btn" title="{{$delete}}"><i class="fa fa-trash" aria-hidden="true"></i></a></td>
+				</tr>
+				{{/foreach}}
+			</tbody>
+		</table>
+	</form>
+</div>
```

Comment:
You can use this template as is for the base template. No need for different theme versions.

Reply:
There are fine differences in - for example - the header size. I copied it from the oauth list.

Reply:
I don't think the differences are meaningful enough to warrant maintaining two different templates.

Reply:
It looks inconsistent.

Reply:
We can polish the kinks in the theme stylesheets, please keep a single template.

Reply:
I just realized that the H-levels aren't consistent over the themes. I guess I will harmonize this. This is especially important for accessibility but also I dislike that chaos.

Reply:
I just realized that the "delete" icon does only work with Frio, so we really need separate templates.

But nonetheless it is not looking great by now.